src/pfe/file-watcher/server/src/projects/projectUtil.ts

```diff
@@ -1172,9 +1225,24 @@ export async function runScript(projectInfo: ProjectInfo, script: string, comman
     const containerName = await getContainerName(projectInfo);
     const logName = getLogName(projectInfo.projectID, projectInfo.location);
     const logDir = await logHelper.getLogDir(projectInfo.projectID, projectInfo.projectName);
-    const args = [projectInfo.location, LOCAL_WORKSPACE, projectID, command, containerName, String(projectInfo.autoBuildEnabled), logName, projectInfo.startMode,
+    const projectName = path.basename(projectInfo.location);
+    let args = [projectInfo.location, LOCAL_WORKSPACE, projectID, command, containerName, String(projectInfo.autoBuildEnabled), logName, projectInfo.startMode,
         projectInfo.debugPort, "NONE", logDir];
 
+    if (projectInfo.projectType == "odo") {
```

Comment:
Why do we need to do this? Does Appsody has equivalent? We should try to avoid burning extension specific code into PFE. Can this be move to part of the extension code?